packs/aws/tests/fixtures/full.yaml

```diff
@@ -0,0 +1,14 @@
+---
+aws_access_key_id: "access_key_id"
+aws_secret_access_key: "secret_key"
+region: "us-west-1"
+interval: 20
+st2_user_data: "/opt/stackstorm/packs/aws/actions/scripts/bootstrap_user.sh"
```

Comment:
Looks like the tests are failing because this config option is specified.

Simply set it to an empty string or none and the tests should pass I believe.